controller/env/appenv.go

```diff
@@ -485,6 +508,15 @@ func (ae *AppEnv) IsAllowed(responderFunc func(ae *AppEnv, rc *response.RequestC
 		}
 
 		responderFunc(ae, rc)
+
+		if !rc.StartTime.IsZero() {
+			timer := ae.GetHostController().GetNetwork().GetMetricsRegistry().Timer(getMetricTimerName(rc.Request))
```

Comment:
Moved here because this function (`IsAllowed`) is only invoked after the REST API has routed the HTTP request to a known handler. Unknown handlers return a 404 before they reach this location.